modules/enhancer.js

```diff
@@ -2,40 +2,39 @@
 
 var resolveStyles = require('./resolve-styles.js');
 var wrapUtils = require('./wrap-utils.js');
+var assign = require('lodash/object/assign');
 
 var enhanceWithRadium = function (ComposedComponent) {
-  var RadiumEnhancer = function () {
-    ComposedComponent.prototype.constructor.call(this);
 
-    if (!this.state) {
-      this.state = {};
-    }
+  class RadiumEnhancer extends ComposedComponent {
+
+    constructor (props) {
```

Comment:
Oh yeah I am totally wrong here. The constructor should take, and pass along, props and context.